20/alpine/Dockerfile

```diff
@@ -1,12 +1,12 @@
 FROM alpine:3.8
 
-ENV OTP_VERSION="20.3.8.14"
-ENV REBAR3_VERSION="3.7.4"
+ENV OTP_VERSION="20.3.8.15"
+ENV REBAR3_VERSION="3.7.5"
```

Comment:
could you use a continuation `\` to concatenate two ENV variables in a single line?

for 21, I have done so last time, you can see the difference by checking the image history, it would be one less layer in the final image:

```console
$ docker image history --no-trunc erlang:21-alpine
IMAGE                                                                     CREATED             CREATED BY
<missing>                                                                 3 days ago          /bin/sh -c #(nop)  ENV OTP_VERSION=21.1.4 REBAR3_VERSION=3.7.5                  0B
``